redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/responses/DatabaseServerV4Response.java

```diff
@@ -0,0 +1,121 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.responses;
+
+// import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.responses.SecretV4Response;
+// import com.sequenceiq.cloudbreak.api.endpoint.v4.workspace.responses.WorkspaceResourceV4Response;
+import com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.base.DatabaseServerV4Base;
+import com.sequenceiq.redbeams.doc.ModelDescriptions;
+import com.sequenceiq.redbeams.doc.ModelDescriptions.DatabaseServer;
+// import com.sequenceiq.cloudbreak.doc.ModelDescriptions.RDSConfigModelDescription;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+@JsonInclude(Include.NON_NULL)
+public class DatabaseServerV4Response extends DatabaseServerV4Base {
+
+    @ApiModelProperty(ModelDescriptions.ID)
+    private Long id;
+
+    // FIXME
+    // @ApiModelProperty(ModelDescriptions.WORKSPACE_OF_THE_RESOURCE)
+    // private WorkspaceResourceV4Response workspace;
+
+    // @ApiModelProperty(RDSConfigModelDescription.CLUSTER_NAMES)
+    // private Set<String> clusterNames;
+
+    // FIXME maybe move to base
+    @ApiModelProperty(value = DatabaseServer.DATABASE_VENDOR, required = true)
+    private String databaseVendor;
+
+    @ApiModelProperty(value = DatabaseServer.DATABASE_VENDOR_DISPLAY_NAME, required = true)
+    private String databaseVendorDisplayName;
```

Comment:
Yeah probably. I based this on DatabaseV4Base and subclasses, which is why I didn't just refactor right away. Now's a good time.

Reply:
Actually, this one should stay just in the response, because its value is internally generated by Cloudbreak - that is, a caller won't ever supply it.